app/src/main/java/com/limelight/Game.java

```diff
@@ -1133,46 +1134,43 @@ protected void onStop() {
             stopConnection();
 
             if (prefConfig.enableLatencyToast) {
-                int averageEndToEndLat = decoderRenderer.getAverageEndToEndLatency();
-                int averageDecoderLat = decoderRenderer.getAverageDecoderLatency();
-                String message = null;
-                if (averageEndToEndLat > 0) {
-                    message = getResources().getString(R.string.conn_client_latency)+" "+averageEndToEndLat+" ms";
-                    if (averageDecoderLat > 0) {
-                        message += " ("+getResources().getString(R.string.conn_client_latency_hw)+" "+averageDecoderLat+" ms)";
-                    }
-                }
-                else if (averageDecoderLat > 0) {
-                    message = getResources().getString(R.string.conn_hardware_latency)+" "+averageDecoderLat+" ms";
-                }
+                // Will format to match `\d+\.\d` (formats 1f as "1.0", 0.12f as "0.1", and 12.34f as "12.3")
+                DecimalFormat decimalFormat = new DecimalFormat("0.0");
+                final String millisecondSuffix = " ms";
```

Comment:
Speaking of localization, I noticed in [someone's Discord video](https://discord.com/channels/352065098472488960/465368546848014336/1202341510721585213) that there are localized versions of the millisecond suffix that we sometimes use and sometimes don't:

![image](https://github.com/moonlight-stream/moonlight-android/assets/33491334/8edadb0b-50c9-4b91-88b8-693c7d6e5107)

I'll see if there is an existing string I can leverage instead of hardcoding it as we have been in this method.

Reply:
Hmm, it appears I'd have to make a new localized string or edit the existing ones to be format strings. Am I allowed to change or add new strings, or is that discouraged because it breaks existing localization? Would it be better for me to surgically steal the localized string for the suffix (like that `мс` in the Russian(?) string above) so that we don't have to get the localizers to do another pass?

Reply:
@cgutman, please let me know which way you'd prefer. The former seems more "correct", but the latter seem easier and probably safe.

Reply:
> I'd have to make a new localized string

@kentyman23 I think this is the correct approach.

@cgutman I'm wondering how the translations are occurring. It seems like it's all manually done through regular PRs? I could set something up on CrowdIn like I do for LizardByte and Sunshine. I have a mono project on CrowdIn that handles all the translations for code projects https://crowdin.com/project/lizardbyte (and another for docs). If you select any language you can see how the translations are divided by GitHub repo and priority. CrowdIn is free for open source. I opted to put everything in one project for ease of management. Let me know if you want me to setup something similar for Moonlight projects.